sky/serve/replica_managers.py

```diff
@@ -195,6 +198,8 @@ def is_scale_down_succeeded(self, initial_delay_seconds: int,
             return False
         if not self.service_ready_now:
             return False
+        if self.preempted:
+            return True
```

Comment:
```suggestion
        if self.preempted:
            return True
        if not self.service_ready_now:
            return False
```
If the `preempted` is set to `True`, then IIUC it is always after a probe failure and thus the `self.service_ready_now` must be `True`?